packages/harmony/src/components/input/TextInput/types.ts

```diff
@@ -55,9 +55,14 @@ export type TextInputProps = Omit<
    */
   startIcon?: IconComponent
   /**
-   * Floating text on the righthand side of the input
+   * Floating icon on the righthand side of the input
    */
   endIcon?: IconComponent
+  /**
+   * @hidden
```

Comment:
interesting with start/end Icon and then endAdornment.... I kinda think we should just have start/end Adornment. curious what you think? not something we need to update, just curious best prop setup

Reply:
talked offline - considering just having start/end adornment as dylan said and if the passed component is an `IconComponent` (i.e. not a react node/element), render it as an icon and otherwise render the component as given. but moving forward with the current change for now until we can discuss w Harmony group